### PR TITLE
docs($controller): deprecate the use of $controllerProvider#allowGlobals

### PR DESCRIPTION
**Pull request by jbedard**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
docs

**Does this PR introduce a breaking change?**
puts the warning for a future one

Anything else that needs to be done to prepare for removing this in 1.7?

### CODE REVIEW

**Comment by petebacondarwin** [comment]
We are going to remove the feature in 1.7?
We should document at which version it will be removed.

**Comment by Narretz** [comment]
I agree. The version when this is removed should go into the deprecated tag. @petebacondarwin can we render this nicely with dgeni?

**Comment by petebacondarwin** [comment]
We should be able to tweak the dgeni templates. I'll take a look tomorrow

File: src/ng/controller.js [hunk 1]
```diff
@@ -59,6 +59,9 @@ function $ControllerProvider() {
    * @ngdoc method
    * @name $controllerProvider#allowGlobals
    * @description If called, allows `$controller` to find controller constructors on `window`
+   *
+   * @deprecated
+   * This method of finding controllers has been deprecated. This will be removed in a future version.
```
Comment (petebacondarwin) [inline src/ng/controller.js]: This will be removed in 1.7

**Comment by jbedard** [comment]
@petebacondarwin updated with a specific version

File: src/ng/controller.js [hunk 2]
```diff
@@ -59,6 +59,9 @@ function $ControllerProvider() {
    * @ngdoc method
    * @name $controllerProvider#allowGlobals
    * @description If called, allows `$controller` to find controller constructors on `window`
+   *
+   * @deprecated
+   * This method of finding controllers has been deprecated. This will be removed in 1.7
```
Comment (gkalpak) [inline src/ng/controller.js]: Missing trailing `.` 😇

Comment (jbedard) [inline src/ng/controller.js]: Updated

**Pull request closed without merging**